### ISSUE TITLE
What should be the loss function when training?

### ISSUE THREAD

**Issue by yangyi0818** [Sanyuan-Chen/CSS_with_Conformer#5]
您好，请问模型训练时的损失函数是怎么设置的呢？微软相关的文章提示说使用能量谱的mse loss作为损失函数（下图），但我在训练的时候发现仅使用这一损失函数，训练的loss是降不下去的，请问是否引入了其他loss或对此loss做了修改？谢谢。
![1638065128(1)](https://user-images.githubusercontent.com/56993487/143726131-d645dbe4-91dc-4db4-ae90-f24d50190067.png)

**Comment by Sanyuan-Chen** [comment]
你好，我们在这篇工作中训练Transformer和Conformer模型时都是只使用基于PIT的MSE loss作为损失函数，没有加入其他的loss项或者进一步的修改，这也是与其他使用PIT方法训练的分离模型是一致的。我们在实验中没有遇到loss未收敛的问题，建议你可以检查PIT的代码实现是否正确，以及一些训练的优化器以及超参数设置（learning rate, weight decay, e.g.）可以参考我们论文中的介绍~

**Comment by whisper0055** [comment]
> 您好，请问模型训练时的损失函数是怎么设置的呢？微软相关的文章提示说使用能量谱的mse loss作为损失函数（下图），但我在训练的时候发现仅使用这一损失函数，训练的loss是降不下去的，请问是否引入了其他loss或对此loss做了修改？谢谢。 ![1638065128(1)](https://user-images.githubusercontent.com/56993487/143726131-d645dbe4-91dc-4db4-ae90-f24d50190067.png)

I have the same problem, did you solve it?
